src/rpc/mining.cpp

```diff
@@ -543,7 +544,10 @@ UniValue getblocktemplate(const JSONRPCRequest& request)
 
         // Create new block
         CScript scriptDummy = CScript() << OP_TRUE;
-        pblocktemplate = BlockAssembler(Params()).CreateNewBlock(scriptDummy, fSupportsSegwit);
+	
+	//TODO: Come back and review how to get drivechain commitments here
+        std::vector<CTxOut> drivechainCommitments;
```

Comment:
Could use help on the best way to get drivechain commitments here. Should we have some sort of specific mechanism to cache `OP_BV` outputs, and then miners can pass the `OP_BV` outputs they wish to redeem in this rpc command? Perhaps we have this function automatically look into an `OP_BV` 'wallet' and select which outputs to add to the coinbase transaction by using some sort of optimizing algorithm (back pack more than likely). Something to talk about tomorrow.